CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## TBD
 ### Added
 - [#1806](https://github.com/org-roam/org-roam/pull/1806) db: support caching and usage of Org 9.5's in-built citations
+- [#1806](https://github.com/org-roam/org-roam/pull/2028) dailies: add `keys` argument to `org-roam-dailies-capture-today` and `org-roam-dailies--capture` functions to give the abilty to get into a capture buffer bypassing the selection screen
```

Comment:
Thanks for trying to update the changelog. I haven't been very diligent with this either, but the PR number is off here.

Reply:
Do you mean it shouldn't be there or it isn't working? 

I've copy-pasted it in the address bar and it works for me:
![image](https://user-images.githubusercontent.com/1290275/149027199-a6319836-7622-4c82-81b5-2e7da67bad72.png)

Reply:
Oh, now I see it - the label itself

Reply:
Fixed that